website/docs/r/managed_cluster.html.markdown

```diff
@@ -0,0 +1,110 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_managed_cluster"
+sidebar_current: "docs-azurerm-resource-managed-cluster"
+description: |-
+  Creates an AKS Managed Cluster instance.
+---
+
+# azurerm\_managed\_cluster
+
+Creates an AKS Managed Cluster instance
```

Comment:
can we expand the acronym here to make this `Azure Managed Kubernetes Cluster (AKS)` or `Azure Kubernetes Service (AKS)`?

Reply:
changed to `Azure Container Service (AKS)` https://docs.microsoft.com/en-us/azure/aks/